woocommerce/navigation
# Use back button component and fix back button styling

Uses the back button component exported in https://github.com/WordPress/gutenberg/pull/25520 to keep styling consistent across nav buttons.

### Screenshot
<img width="308" alt="Screen Shot 2020-09-21 at 3 10 45 PM" src="https://user-images.githubusercontent.com/10561050/93825631-53c14a80-fc6e-11ea-90aa-f8371d488728.png">


### Testing

1. `npm link && npm run build` the branch from https://github.com/WordPress/gutenberg/pull/25520 to this repo.
1. Make sure that the back buttons at the root level and all sub-levels are styled appropriately.
1. Check that back buttons all work as expected with no console errors.
